packages/js/components/src/experimental-select-control/select-control.tsx

```diff
@@ -84,14 +84,16 @@ function SelectControl< ItemType = DefaultItemType >( {
 }: SelectControlProps< ItemType > ) {
 	const [ isFocused, setIsFocused ] = useState( false );
 	const [ inputValue, setInputValue ] = useState( '' );
+	const inputRef = useRef< HTMLInputElement >( null );
```

Comment:
Do you think it would be possible to move this into the `ComboBox` component instead?
Given we are only making use of it there aside from passing it into `getDropdownProps`?

Maybe we could move those function calls to the `ComboBox` component as well? Or we could merge the refs in the `ComboBox` component something like:
```
function mergeRefs(...refs) {
    refs.forEach((inputRef) => {
      if (typeof inputRef === 'function') {
        inputRef(ref);
      } else {
        inputRef.current = ref;
      }
    });
  };
const Combobox = ( { inputProps } ) => { ....

   const inputRef = useRef< HTMLInputElement >( null );
   // ....
   return <input ref={ mergeRefs(inputRef, inputProps.ref ) } ..../>
```

Reply:
The issue with moving this into combo box, would be increasing the number of props and knowledge inside `ComboBox` pretty extensively.

I like the idea of merging refs, though I don't think we have any utility yet like `mergeRefs` in WC.  I did this manually in https://github.com/woocommerce/woocommerce/pull/34788/commits/f6be8d6a7e6dbaf1dbc3597621960d298c6f2629 though it was a little weird since I believe the ref from `inputProps` is mistyped.  

Seems to be working okay on my end.  Could you smoke test as well and make sure you don't notice any odd behavior?

Reply:
> Seems to be working okay on my end. Could you smoke test as well and make sure you don't notice any odd behavior?

Yeah it seems to behave correctly as far as I can tell